Modules/Core/Common/include/itkConicShellInteriorExteriorSpatialFunction.hxx

```diff
@@ -94,14 +94,14 @@ ConicShellInteriorExteriorSpatialFunction<VDimension, TInput>::PrintSelf(std::os
 
   unsigned int i;
   os << indent << "Origin: [";
-  for (i = 0; i < VDimension - 1; ++i)
+  for (i = 0; i + 1 < VDimension; ++i)
```

Comment:
Do we allow Dimension to be zero? If we don't, this is not needed.

Reply:
Yes, I think that _**all**_ of the changes in this pull request are not needed!  I mean that in the senses that either
1. the variable's range precludes the underflow, or
2. the loop is only entered after a runtime check that the variable is in a range that will not underflow.

The added value of this pull request comes about only as insurance: if we later allow `VDimension` to be zero or, in other scenarios, if we later remove the runtime range check.

I am now convinced that this isn't enough added value ... and will close this pull request.